ERCS/erc-7660.md

```diff
@@ -0,0 +1,330 @@
+---
+eip: 7660
+title: Release tokens on predefined periods
+description: Tokens can be linearly released based on the predefined period.
+author: Ali Kaya (@AliKaya) <ali.kayatur747@gmail.com>
+discussions-to: https://ethereum-magicians.org/t/erc-7660-a-protocol-for-erc20-tokens-which-released-based-on-predefined-periods-controlled/19305
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-03-08
+requires: 20
+---
+
+## Abstract
+
+This proposal outlines [ERC-20](./eip-20.md) tokens which released based on predefined periods,it uses a novel "Sorter" mechanism in this protocol, which automatically consolidates multiple transfers in smart contracts. This feature ensures token release based on different transfer times while minimizing gas fees significantly.
+
+## Motivation
+
+Rely on this new standard to establish a Token category with a consensus period by default (passive), rather than the motivation and means of linear release (subjective).
```

Comment:
Could you define the terms you're using here? Specifically "consensus period", "passive", "linear release" and "subjective". I'm not an expert on tokens, so I probably represent the average reader of ERCs pretty well, and I have no idea what you're talking about.